introduction_to_amazon_algorithms/jumpstart_image_classification/Amazon_JumpStart_Image_Classification.ipynb

```diff
@@ -2,15 +2,15 @@
  "cells": [
```

Comment:
<p>Typo: <code>fientuning</code> -&gt; <code>finetuning</code></p><p> </p><p>Add a space before the left paren in <code>dataset(old or new)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3429/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>Done.</p>